uaclient/tests/test_status.py

```diff
@@ -536,20 +530,17 @@ def test_nonroot_unattached_is_same_as_unattached_root(
         m_get_available_resources.return_value = [
             {"name": "esm-infra", "available": True}
         ]
-        m_getuid.return_value = 1000
-        cfg = FakeConfig()
+        cfg = FakeConfig(root_mode=False)
         nonroot_status = status.status(cfg=cfg)
 
-        m_getuid.return_value = 0
+        cfg = FakeConfig(root_mode=True)
         root_unattached_status = status.status(cfg=cfg)
 
         assert root_unattached_status == nonroot_status
 
     @mock.patch("uaclient.status.get_available_resources")
-    @mock.patch("uaclient.status.os.getuid")
     def test_root_followed_by_nonroot(
```

Comment:
I think this whole test can just be removed, because its purpose was to test that the nonroot used the cache. Right? @lucasmoura @CalvoM